.github/workflows/performance_metrics_cron.yml

```diff
@@ -106,10 +106,10 @@ jobs:
         dotnet-version:
           - 3.1.301
```

Comment:
Note: It looks like this will need to be bumped to .NET 6 as part of fixing `TestAccAwsCsS3Folder` and `TestAccAwsFsS3Folder` for #1430 due to:

https://github.com/pulumi/examples/actions/runs/5090579201/jobs/9149574791#step:15:77

```
error NU1202: Package Pulumi.Aws 5.41.0 is not compatible with netcoreapp3.1 (.NETCoreApp,Version=v3.1). Package Pulumi.Aws 5.41.0 supports: net6.0 (.NETCoreApp,Version=v6.0)
```

I'll do that as part of #1430